src/cloudant/account.py

```diff
@@ -27,14 +27,16 @@ class Cloudant(dict):
 
     Optional parameters can be passed to control behaviour:
 
-    :param cloudant_url: Fully qualified https:// url for the cloudant service,
-      if not provided defaults to https://<username>.cloudant.com
+    :param cloudant_url: Fully qualified https:// url for the
+      cloudant service, if not provided defaults to
+      https://<username>.cloudant.com
 
-    :param x_cloudant_user: Override the X-Cloudant-User setting used to auth. This
-      is needed to auth on someones behalf, eg with an admin account
+    :param x_cloudant_user: Override the X-Cloudant-User setting
```

Comment:
This is a little misleading. The BA creds or cookie is the auth part, the x-cloudant-user is the other part (which is hard to put into words) "the account being manipulated"